src/Service/Requests/UpdateSynonymRule.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace SilverStripe\ForagerBifrost\Service\Requests;
+
+use Elastic\EnterpriseSearch\AppSearch\Request\CreateSynonymSet;
+use SilverStripe\Forager\Service\Query\SynonymRule as SynonymRuleQuery;
+use SilverStripe\ForagerBifrost\Processors\SynonymRuleProcessor;
+use stdClass;
+
+/**
+ * Even though the Elastic Enterprise Search SDK request is called PUT, it's actually a PATCH, as it requires the
```

Comment:
this is a bit confusing as the method type is still PUT? was it mean to be patch? our api is PUT so are we kinda throwing shade on our own api (which incidentally says that the PUT endpoint is for either create or update....)

Reply:
I'll just remove this comment. Our API does support create and update through the `PUT` endpoint though - App Search does not.